doc/src/challenges/0011-floats-ints.md

```diff
@@ -0,0 +1,55 @@
+# Challenge 11: Safety of Floats and Integers in `core::num`
+
+
+- **Status:** Open
+- **Tracking Issue:** [Link to issue](https://github.com/model-checking/verify-rust-std/issues/59)
+- **Start date:** *2024-08-20*
+- **End date:** *2024-12-10*
+
+-------------------
+
+## Goal
+
+Verify the safety of public unsafe methods for floats and integers in `core::num`.
+
+## Success Criteria
+
+### Signed Integers
+
+Prove the absence of undefined behavior in the following methods:
+
+1. `int_macros::unchecked_add`
```

Comment:
I would use something like `<INT_TYPE>::unchecked_add`

Reply:
I decided to go with a table, since a whole list of `<INT_TYPE>::<method` made the text long and harder to parse.